src/pfe/file-watcher/server/test/functional-test/configs/pfe.config.ts

```diff
@@ -17,7 +17,7 @@ const pfe = {
 
 export const baseURL = pfe.PROTOCOL + "://" + pfe.HOST + ":" + pfe.PORT;
 
-export const deploymentRegistry = process.env.IN_K8_REGISTRY || "codewindtest"; // need to change the default registry to the fallback of the che registry installed for test
+export const deploymentRegistry = "docker-registry.default.svc:5000/eclipse-che";
```

Comment:
I think we can update the `project/namespace` for the deployment registry with this https://github.com/eclipse/codewind-docs/blob/master/docs/_documentations/openshiftregistry.md as we move our Test cluster Che to `7.0.0`, then we wont have the problem of the char limit on the Che Administration Tab

Reply:
I think we can keep the deployment registry a default value of `docker-registry.default.svc:5000/eclipse-che`. What do you think?

Reply:
sure, if we can use that namespace/project and the token without expriing, we can use that... I think we can use the token from sa `che` like mentioned in the doc

```
maysun$ k describe sa che
Name:                che
Namespace:           eclipse-che
Labels:              app=che
                     template=che
Annotations:         openshift.io/generated-by: OpenShiftNewApp
Image pull secrets:  che-dockercfg-l9wsj
Mountable secrets:   che-dockercfg-l9wsj
                     che-token-jbqn9
Tokens:              che-token-8k52s
                     che-token-jbqn9
Events:              <none>
```

Reply:
Yup